src/main/java/frc/robot/commands/Presets/AmpPreset.java

```diff
@@ -2,14 +2,15 @@
 
 
 import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.PivotSubsystem;
+
 import static frc.robot.RobotContainer.PIVOT;
 
 public class AmpPreset extends Command{
-    public AmpPreset(){
+    public AmpPreset(PivotSubsystem PIVOT){
```

Comment:
This is still working importing the static

The idea is that the subsystem is pass to this command from robotContainer.

For sake of time please restore it to how it was before.
I can explain the best practice later. Same for the other commands

Reply:
Ok